api/Navigator.json

```diff
@@ -548,10 +548,24 @@
               "notes": "Not supported on macOS, see <a href='https://crbug.com/1144920'>bug 1144920</a>."
             },
             "firefox": {
-              "version_added": false
+              "version_added": "96",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#dom.webshare.enabled",
+                  "value_to_set": "enabled"
+                }
+              ]
             },
             "firefox_android": {
-              "version_added": false
+              "version_added": "96",
+              "flags": [
```

Comment:
[dom.webshare.enabled=true](https://searchfox.org/mozilla-central/rev/4c184ca81b28f1ccffbfd08f465709b95bcb4aa1/mobile/android/app/geckoview-prefs.js#37) on Firefox Android as default even if on release channel. It is unnecessary to this `flags` entry. You should remove this `flags` value.

 [Firefox desktop is still false](https://searchfox.org/mozilla-central/rev/4c184ca81b28f1ccffbfd08f465709b95bcb4aa1/modules/libpref/init/StaticPrefList.yaml#4137-4144) on stable channel.

Reply:
Thanks very much. I will update this on Friday!